lib/index.ts

```diff
@@ -11,6 +11,19 @@ export function removeHtmlComments(str: string): string {
   return str.replace(/<!--[\s\S]*?(-->|$)/g, "");
 }
 
+/**
+ * Extracts the inner html of every element inside the head
+ * @param {String} code a HTML string of the head
+ * @returns {String} the inner html of every element in the head or an empty string if no head is found
+ */
+
+export function extractHTMLElement(code: string, tag: string = "head"): string {
+  const expression = new RegExp(
+    "(?<=<" + tag + "\\s*>)(?:.|\\s*)*?(?=<\\/" + tag + "\\s*>)"
+  );
+  return code.match(expression)?.toString() ?? "";
```

Comment:
The description, params, and def do not match.

Is this just for `head` elements? By the look of things, this is for all **non**-void elements provided they have no attributes.

Reply:
It was originally designed for that but I wanted it to be more useful so that I can justify adding it to the library.